Platon/backend/app/follow_system/models.py

```diff
@@ -34,3 +34,14 @@ def __init__(self, owner_id, commented_user_id, rate, text):
         self.commented_user_id = commented_user_id
         self.rate = rate
         self.text = text
+
+class Reports(db.Model):
+    id = db.Column(db.Integer,primary_key=True,autoincrement=True)
+    owner_id = db.Column(db.Integer,db.ForeignKey('users.id',ondelete="CASCADE"))
+    reported_user_id = db.Column(db.Integer,db.ForeignKey('users.id',ondelete="CASCADE"))
```

Comment:
It is not a really big issue but can it be better if these reports will not be deleted if these users are removed from the system?  I want to know your thoughts about this.

Reply:
If the users are removed from the system, what should owner-id and reported-user-id columns be then?  It can be misleading to hold id's that are not in the database. I am open to suggestions.

Reply:
You can set them to NULL bu chaning the on_delete constraint of the Foreign Keys. In SQL it is SET NULL, but I do not know the SQLAlchemy version of it.

Reply:
I am okay to both of these solutions so I will wait for other feedbacks and change them to null if no one refuses or has better idea.

Reply:
@cakmadam98 @alperendivriklioglu  we can use your opinions about this issue.

Reply:
I think there is no need to change the current version. Setting those fields to NULL is a solution, but without those fields Report instance would not make sense. Therefore, deleting them is okay in my opinion.